OpenRA.Game/Exts.cs

```diff
@@ -151,6 +151,26 @@ static T Random<T>(IEnumerable<T> ts, MersenneTwister r, bool throws)
 				return xs.ElementAt(r.Next(xs.Count));
 		}
 
+		public static Rectangle UnionRectangles(this IEnumerable<Rectangle> rects)
```

Comment:
The `Rectangles` is implicit from it acting on `IEnumerable<Rectangle>`. Can we call this just `Union` instead to match `Product` and `SymmetricDifference` below?

Reply:
Renamed.